.ci/verify_test_api_requests.py

```diff
@@ -27,29 +27,50 @@
 def verify_test_case(cli_arguments, path):
     print("Verifying test case: '%s'." % (path))
 
-    import_module_name, arguments, expected, output_modifier = tests.api.test_api.get_api_test_info(path)
+    import_module_name, arguments, expected, is_error, output_modifier = tests.api.test_api.get_api_test_info(path)
 
     for key, value in vars(cli_arguments).items():
         if ((value is not None) or (value)):
             arguments[key] = value
 
     api_module = importlib.import_module(import_module_name)
 
-    actual = api_module.send(arguments)
+    try:
+        actual = api_module.send(arguments)
+
+        if (is_error):
+            _format_error(expected, actual, path, "Test case does not raise an error when one was expected")
+
```

Comment:
Remove this newline.
Be careful, having two disjoint single lines can be visually confusing (especially in a nested block).
Regardless, in this case they belong together.

You also have this issue elsewhere.